source/common/develop/ide/tools/ghapi.rst

```diff
@@ -41,9 +41,9 @@ In your repo go to **Settings -> Secrets** and create Client ID and Secret ID co
    :alt: GH secret
 
 
-To create your secrets for the Codespace (ie applies for all repos in the account and available for all collaborators), click on **Actions-> Codespaces** or to create just for the individual repo, click on **New repository secret** and create your secret keys.
+To create your secrets for your organisations repo (ie applies for all repos in the account and available for all collaborators), see `encrypted secrets for an organization <https://docs.github.com/en/actions/security-guides/encrypted-secrets#creating-encrypted-secrets-for-an-organization/>`_ or to create just for the individual repo, click on **New repository secret** and create your secret keys.
```

Comment:
To create secrets for your organization's repo (these will apply to all repos in the account and be available for all collaborators), see `encrypted secrets for an organization <https://docs.github.com/en/actions/security-guides/encrypted-secrets#creating-encrypted-secrets-for-an-organization/>`_. To create these for an individual repo, click on **New repository secret** and create the secret keys.